mobile/native/src/ln_dlc/mod.rs

```diff
@@ -95,7 +96,9 @@ pub async fn refresh_wallet_info() -> Result<()> {
 }
 
 pub fn get_seed_phrase() -> Vec<String> {
-    NODE.get().get_seed_phrase()
+    SEED.try_get()
+        .expect("SEED to be initialised")
```

Comment:
:upside_down_face: May as well use `get` here then.

Reply:
I did this on purpose to have a more clear error message to readability if it ever happens.
 the `"used set() before get()"` with a line number is IMHO far worse than this when we have more than one thing that can print this ☺️